extensions/EXT/EXT_texture_query_lod.txt

```diff
@@ -0,0 +1,183 @@
+Name
+
+    EXT_texture_query_lod
+
+Name Strings
+
+    GL_EXT_texture_query_lod
+
+Contact
+
+    Gert Wollny (gert wollny 'at' collabora.com)
+
+Contributors
+
+    Pat Brown, NVIDIA
+    Greg Roth, NVIDIA
+    Eric Werness, NVIDIA
+
+Notice
+
+    Copyright (c) 2019 Collabora LTD 
+    Copyright (c) 2009-2013 The Khronos Group Inc. Copyright terms at
+        http://www.khronos.org/registry/speccopyright.html
+
+Status
+
+    FIXME
+
+Version
+
+    Last Modified Date:         04/02/2019
+    Revision:                   1
+    Based on ARB_texture_query_lod version 4, modified 2013/10/04.
+
+Number
+
+    OpenGL ES extension #310
+
+Dependencies
+
+    OpenGL ES 3.0 is required.
+
+    OpenGL Shading Language 3.00 ES is required
+
+    EXT_gpu_shader4 is required.
```

Comment:
There so such extension (for ES) - and I don't think this dependency is needed as long as you already require ESSL 3.0.

Reply:
Indeed, fix pushed